extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -252,9 +254,6 @@ describes.realWin('amp-list component', {
       element.setAttribute('src', 'https://new.com/list.json');
       list.mutatedAttributesCallback({'src': 'https://new.com/list.json'});
     });
-    listMock.expects('toggleLoading').withExactArgs(false).once();
-    listMock.expects('togglePlaceholder').withExactArgs(false).once();
```

Comment:
these `listMock`s were set twice: once in `expectFetchAndRender` and once in the test itself, but they are only called once. when `rendered_()` returns a promise (`measurePromise`), these conditions fail.